apps/easypid/src/locales/pt/messages.json

```
@@ -0,0 +1,3906 @@
+		},
+		"comments": ["Used after card count to indicate the total number"],
+		"origin": [["src/features/wallet/components/AllCardsCard.tsx", 23]],
+		"translation": "{0} cartões no total"
+	},
+	"activity.sharedSummaryPlural": {
+		"message": "{amountShared} credentials were shared.",
+		"placeholders": {
+			"amountShared": ["amountShared"]
+		},
+		"comments": ["Shown when multiple credentials were successfully shared"],
+		"origin": [["src/features/activity/FunkeActivityDetailScreen.tsx", 218]],
+		"translation": "{amountShared} credenciais foram compartilhadas."
+	},
+	"inbox.notificationType.fromLabel": {
+		"message": "{baseLabel} from {0}",
+		"placeholders": {
+			"0": ["notification.contactLabel"],
+			"baseLabel": ["baseLabel"]
+		},
+		"comments": ["Label like \"Card from X\" or \"Request from Y\""],
+		"origin": [["src/features/didcomm/InboxScreen.tsx", 91]],
+		"translation": "{baseLabel} de {0}"
+	},
+	"dateFormatting.mulitpleDays": {
+		"message": "{days} days",
+		"placeholders": {
+			"days": ["days"]
+		},
+		"comments": [],
+		"origin": [["../../packages/utils/src/format.ts", 105]],
+		"translation": "{days} dias"
+	},
+	"requestedAttributes.subheading": {
+		"message": "{disclosedAttributeLength} from {0}",
+		"placeholders": {
+			"0": ["activeCredential.display.name"],
+			"disclosedAttributeLength": ["disclosedAttributeLength"]
+		},
+		"comments": ["Subheading under attribute list: X from Y"],
+		"origin": [
+			["src/features/share/FunkeRequestedAttributesDetailScreen.tsx", 138]
+		],
+		"translation": "{disclosedAttributeLength} de {0}"
+	},
+	"credentialCardSlide.subtitle.default": {
+		"message": "{issuerName} wants to issue you the following card:",
+		"placeholders": {
+			"issuerName": ["issuerName"]
+		},
+		"comments": [
+			"Subtitle shown when the issuer wants to give the user a card with no auth required"
+		],
+		"origin": [["src/features/receive/slides/CredentialCardSlide.tsx", 44]],
+		"translation": "{issuerName} quer emitir o seguinte cartão para ti:"
+	},
+	"verifyPartySlide.connectSubtitle": {
+		"message": "{name} wants to connect with you.",
+		"placeholders": {
+			"name": ["name"]
+		},
+		"comments": [],
+		"origin": [["src/features/receive/slides/VerifyPartySlide.tsx", 153]],
+		"translation": "{name} quer se conectar contigo."
+	},
+	"verifyPartySlide.signingSubtitle": {
+		"message": "{name} wants to interact to create a digital signature for a document.",
+		"placeholders": {
+			"name": ["name"]
+		},
+		"comments": [],
+		"origin": [["src/features/receive/slides/VerifyPartySlide.tsx", 131]],
+		"translation": "{name} quer interagir para criar uma assinatura digital para um documento."
+	},
+	"verifyPartySlide.offerCardSubtitle": {
+		"message": "{name} wants to offer you a card.",
+		"placeholders": {
+			"name": ["name"]
+		},
+		"comments": [],
+		"origin": [["src/features/receive/slides/VerifyPartySlide.tsx", 121]],
+		"translation": "{name} quer oferecer um cartão para ti."
```

Many places with "para ti" sound very weird. Either remove, or use the "-lhe" or "-te".

```suggestion
		"translation": "{name} quer-lhe oferecer um cartão."
```